### ISSUE TITLE
`DeprecationWarning` invalid escape sequence

### ISSUE THREAD

**Issue by johnthagen** [manatlan/vbuild#10]
```
vbuild/__init__.py:230: DeprecationWarning: invalid escape sequence '\*'
  css = re.sub(re.compile("/\*.*?\*/", re.DOTALL), "", css)
```

https://github.com/manatlan/vbuild/blob/6b266fd20b57162d72eea5cbab403b0daaff19d8/vbuild/__init__.py#L230

This is being promoted to a `SyntaxWarning` in Python 3.12:

- https://docs.python.org/3.12/whatsnew/3.12.html#other-language-changes

> A backslash-character pair that is not a valid escape sequence now generates a [SyntaxWarning](https://docs.python.org/3.12/library/exceptions.html#SyntaxWarning), instead of [DeprecationWarning](https://docs.python.org/3.12/library/exceptions.html#DeprecationWarning). For example, re.compile("\d+\.\d+") now emits a [SyntaxWarning](https://docs.python.org/3.12/library/exceptions.html#SyntaxWarning) ("\d" is an invalid escape sequence), use raw strings for regular expression: re.compile(r"\d+\.\d+"). In a future Python version, [SyntaxError](https://docs.python.org/3.12/library/exceptions.html#SyntaxError) will eventually be raised, instead of [SyntaxWarning](https://docs.python.org/3.12/library/exceptions.html#SyntaxWarning). (Contributed by Victor Stinner in [gh-98401](https://github.com/python/cpython/issues/98401).)

This will break in a future Python release when this is promoted to a `SyntaxError`.

**Comment by johnthagen** [comment]
Related:

- #9

**Comment by manatlan** [comment]
fixed in 0.8.2
